services/src/atdd/src/test/resources/features/mvp.1/TopologyEventsBasic.feature

```diff
@@ -31,7 +31,7 @@ Feature: Basic Topology Events
     When the controller learns the topology
     And multiple links exist between all switches
     And a link is added in the middle
-    Then the link appears in the topology engine.
+    Then the link appears in the topology engine in 60 seconds.
```

Comment:
I thought the timeout only applies to the ISL outage, not discovery. Links usually appear right away